spec/index.html

```diff
@@ -495,6 +498,31 @@ <h2>RDF Documents and Syntaxes</h2>
     and still the subject of group discussion. An alternative to conformance
     levels, "profiles", may be adopted instead, abandoned, or described in 
     another specification.</p>
+
+  <section id="rdf-strings">
+    <h2>Strings in RDF</h2>
+
+    <p>RDF uses Unicode [[Unicode]] as the fundamental representation for string values.
+      Within this, and related specifications, the term <dfn id="dfn-rdf-string">string</dfn>,
+      or <a data-lt="string">RDF string</a>,
+      is used to describe an ordered sequence of zero or more
+      <a data-lt="code point" class="lint-ignore">Unicode code points</a>
+      which are <a data-cite="i18n-glossary#dfn-scalar-value" class="lint-ignore">Unicode scalar values</a>.
+      Unicode scalar values do not include the
+      <a data-cite="i18n-glossary#dfn-surrogate" class="lint-ignore">surrogate code points</a>.
+      Note that most <a>concrete RDF syntaxes</a> require the use
+      of the UTF-8 character encoding [[!RFC3629]], 
+      which requires that certain non-character values be expressed
+      using the `\u0000` or `\U00000000` forms.
```

Comment:
the UTF-8 character encoding doesn't define escape sequeneces. These would be JS escapes. Other formats use a different format, such as HTML/XML's NCRs (`&#x20ac;`).

Reply:
I think it is OK because it's covered by the text "Note that most concrete RDF syntaxes" at the start of the sentence.
The RDF formats Turtle, TriG, N-triples and N-quads, and SPARQL, all have `\u0000` and `\U00000000`.

Reply:
Fair enough, although the sentence actually says that UTF-8 requires this ;-).